plugins/CoreAdminHome/Tasks.php

```diff
@@ -101,6 +105,19 @@ public function optimizeArchiveTable()
     }
 
     /**
+     * Update the referrer spam blacklist
+     *
+     * @see https://github.com/piwik/referrer-spam-blacklist
+     */
+    public function updateSpammerBlacklist()
+    {
+        $url = 'https://raw.githubusercontent.com/piwik/referrer-spam-blacklist/master/spammers.txt';
+        $list = Http::sendHttpRequest($url, 10);
+        $list = preg_split("/\r\n|\n|\r/", $list);
```

Comment:
add some sanity check that list has at least 100 spammers or so (in case we screw the list on master, or in case the file was not downloaded correctly)